### PR TITLE
Remove HSV and YCBCR references in wrong value range

### PR DESCRIPTION
**Pull request by alessiamarcolini**
## Description

Remove tests, fixtures and expectations with images in HSV and YCBCR wrongly converted to RGB.
This doesn't impact the codebase, only the tests.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [x] Enhancement/optimization
- [ ] Documentation

## Issues Fixed or Closed by This PR

* Fixes: 

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [CONTRIBUTING](https://github.com/histolab/histolab/blob/master/CONTRIBUTING.md) document.
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Review by ernestoarbitrio — APPROVED** [review]
LGTM but I would wait for the CI before merging it

**Comment by alessiamarcolini** [comment]
@ernestoarbitrio I see now that it's running, great

**Pull request merged**
